elastic/beats
# Enable netinfo in add metadata processors

## What does this PR do?

Enable netinfo in add_host_metadata and add_observer_metadata processors by default.

## Why is it important?

This network information metadata is used in observability solutions.

This change shouldn't break backwards compatibility or produce any conflict as the added fields are in ECS. 

## Checklist


## How to test this PR locally

* Run a beat with the `add_host_metadata` processor without any options.
* Check that all events contain `host.ip` and `host.mac` fields.

## Related issues

- Superseds elastic/beats#15347
